backend/api/v2beta1/pipeline.proto

```diff
@@ -125,10 +124,11 @@ service PipelineService {
   rpc ListPipelineVersions(ListPipelineVersionsRequest)
       returns (ListPipelineVersionsResponse) {
     option (google.api.http) = {
-      get: "/apis/v2beta1/pipeline_versions"
+      get: "/apis/v2beta1/pipelines/{pipeline_id}/versions"
```

Comment:
nit: maybe move the `DeletePipelineVersion` to the last rpc definition, to follow CRUD order?